GetandTake/Program.cs

```diff
@@ -5,7 +5,12 @@
 var builder = WebApplication.CreateBuilder(args);
 
 // Add services to the container.
-builder.Services.AddRazorPages();
+builder.Services.AddRazorPages().AddRazorRuntimeCompilation().AddMvcOptions(options =>
```

Comment:
For such long chains of methods calls I advise to always put them in column, like:
```suggestion
builder.Services
        .AddRazorPages()
        .AddRazorRuntimeCompilation()
        .AddMvcOptions(options =>
        {
        });
```

It would simplify reading